addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -370,6 +370,57 @@ options.Class.include({
             },
         });
     },
+    /**
+     * @private
+     * @param {String} type either 'head' or 'footer'
```

Comment:
Review your doc: not private, wrong param :wink: Since the parameters are the same as the standard selectClass method, I often write `@see this.selectClass for parameters`